packages/eas-cli/src/credentials/ios/appstore/provisioningProfileAdhoc.ts

```diff
@@ -224,7 +224,7 @@ export async function createOrReuseAdhocProvisioningProfileAsync(
   bundleIdentifier: string,
   distCertSerialNumber: string
 ): Promise<ProvisioningProfile> {
-  const spinner = ora(`Handling Adhoc provisioning profiles on Apple Developer Portal...`).start();
+  const spinner = ora(`Handling Apple AdHoc provisioning profiles`).start();
```

Comment:
```suggestion
  const spinner = ora(`Handling Apple ad hoc provisioning profiles`).start();
```

https://developer.apple.com/library/archive/documentation/ToolsLanguages/Conceptual/DevPortalGuide/DistributinganApp/DistributinganApp.html